test/controllers/events_controller_test.rb

```diff
@@ -147,6 +161,27 @@ def setup_live_events
       assert_match("data-name='w'  selected", @response.body)
     end
 
+    test "selects correct possible aggregation for 3m for month" do
+      setup_live_events
+      sign_in @user
+      get project_event_path(@project.name, @event.name.parameterize, params: { agg: "m", date: "3m" })
+      assert_match("data-name='m'  selected", @response.body)
+    end
+
+    test "selects correct possible aggregation for 3m for year" do
+      setup_live_events
+      sign_in @user
+      get project_event_path(@project.name, @event.name.parameterize, params: { agg: "y", date: "3m" })
+      assert_match("data-name='y'  selected", @response.body)
```

Comment:
Shoudn't it disallow year aggregation for `3m` date range?

Reply:
at the moment, the cut off is at one month, after that, week, month and year can be selected, do you want to increase it to 3 months or more?